src/sequelize/migrations/20190222135632-create-kubelet-access-token.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('KubeletAccessTokens', {
+      id: {
+        type: Sequelize.INTEGER,
+        primaryKey: true,
+        autoIncrement: true,
+        allowNull: false,
+        field: 'id'
+      },
+      expirationTime: {
+        type: Sequelize.BIGINT,
+        field: 'expiration_time'
+      },
+      token: {
+        type: Sequelize.TEXT,
+        field: 'token'
+      },
+      iofogUuid: {
```

Comment:
`vk-token` is per user and should not be associated with an ioFog node.

Reply:
fix